addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -7,6 +7,7 @@ import checkoutForm from 'payment.checkout_form';
 checkoutForm.include({
     events: _.extend({}, checkoutForm.prototype.events || {}, {
         'change .o_wpayment_fee_impact': '_onFeeParameterChange',
+        'focus .o_wpayment_fee_impact': '_updateAmount',
```

Comment:
If we do that, the fee might not match the chosen amount.
E.g. if the fee of the payment acquirer inovolves a percentage of the amount.